docs/branching-and-tagging.md

```diff
@@ -61,19 +61,19 @@ via Spark SQL.
 snapshots will be kept, and the branch reference itself will be retained for 1 week. 
 ```sql
 -- Create a tag for the first end of week snapshot. Retain the snapshot for a week
-ALTER TABLE prod.db.table CREATE TAG 'EOW-01' AS OF VERSION 7 RETAIN 7 DAYS
+ALTER TABLE prod.db.table CREATE TAG EOW-01 AS OF VERSION 7 RETAIN 7 DAYS
```

Comment:
I don't think you can use `-` in an identifier name. Shouldn't this be quoted?

Reply:
You are right but it should be quoted by '`'.